src/main/java/com/rtb/projectmanagementtool/project/ProjectData.java

```diff
@@ -0,0 +1,204 @@
+/**
+ * ProjectData.java - this file implements the Project class, which contains details about a
+ * particular project, including the members and tasks within it.
+ */
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.HashSet;
+
+enum User {
+  CREATOR,
+  ADMIN,
+  REGULAR;
+}
+
+public class ProjectData {
+  /**
+   * constant variables used to specify the names of each property of the Project entity. Used to
+   * minimize redundancy and also make renaming properties easy if needed
+   */
+  private final String PROPERTY_NAME = "name";
+
+  private final String PROPERTY_DESCRIPTION = "description";
+  private final String PROPERTY_USER_IDS = "userIds";
+  private final String PROPERTY_TASK_IDS = "taskIds";
+
+  private DatastoreService datastore;
```

Comment:
See note I had above regarding limiting the number of "connections" you have open.  Another thing to think about, you'll have multiples of this ProjectData items and the DatastoreService object most probably is pretty heavy.  We wouldn't want to use up resources to have multiples of these